nservicebus/azure-servicebus/multiple-namespaces-support.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Multiple namespace support
+summary: Configuring Azure ServiceBus transport to support different partitioning strategies. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+reviewed: 2016-04-18
+---
+
+// INTRODUCTION HERE!!!
+
+To configure _namespace partitioning strategy_ properly users have to select desired `INamespacePartitioningStrategy` implementation and map related namespaces, as showed in the next snippets, for each supported strategies.
+
+### Single namespace partitioning #
+
+The default _namespace partitioning strategy_ is `SingleNamespacePartitioning`. With the default setup, transports uses only a single namespace to send and receive messages.  
+Users can provide namespace connection string directly with `ConnectionString` extension method:
```

Comment:
> Users...
What if they are customers? :)

An option:

Namespace connection string can be defined via code using `ConnectionString` extension: